plugins/aea-cli-ipfs/aea_cli_ipfs/ipfs_utils.py

```diff
@@ -324,29 +324,28 @@ def download(
         :return: downloaded path
         """
 
-        def move_to_target_dir(download_tmp_path) -> str:
+        def move_to_target_dir(download_tmp_path: Path) -> str:
             """Move downloaded content to target directory"""
             if download_tmp_path.is_file():
-                # file is ok!
+                # Downloaded object is a file and can be moved easily
                 shutil.move(str(download_tmp_path), str(target_dir))
                 return str(target_dir)
 
             if fix_path:
                 # identify name of package
                 paths = list(download_tmp_path.glob("*"))
-                assumption_is_valid = len(paths) == 1 and paths[0].is_dir()
-                if not assumption_is_valid:  # pragma: no cover
-                    error_msg = f"Expected a single directory, found: {paths}"
-                    raise DownloadError(error_msg)
```

Comment:
The downloaded object can be either a file or a directory, which can be wrapped under a wrapper node. So this gives 4 possibilities for a downloaded object. A file, file wrapped, dir and dir wrapped. We need a better mechanism to detect these 4 and filter out aea component packages. https://github.com/valory-xyz/open-aea/issues/519